service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/CafcassEmailContentProviderSDOIssuedTest.java

```diff
@@ -37,7 +38,7 @@ void shouldReturnNotifyData() {
             .leadRespondentsName("Smith")
             .hearingDate("1 January 2020")
             .reference(caseData.getId().toString())
-            .caseUrl(caseUrl(caseData.getId().toString(), "OrdersTab"))
+            .caseUrl(caseUrl(caseData.getId().toString(), ORDERS))
```

Comment:
definitely dislike, using production code to generate expected data is not good, if the production generation method was accidentally changed then you wouldn't be able to catch the change here